src/net/db.rs

```diff
@@ -1532,6 +1933,103 @@ mod test {
         }
     }
 
+    #[test]
+    fn test_insert_or_replace_stacker_dbs() {
```

Comment:
For each of these new tests, can you add a comment describing what the test is intending to cover? What is the scenario being tested? Just like a sentence or two about what the test is supposed to be orchestrating would be helpful when figuring out what situation are being covered.